src/Chromosome.java

```diff
@@ -1,9 +1,103 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 public class Chromosome {
 
     public double[][] adjacencyMatrix;
 
-    public Chromosome(List<Double> dna) {}
+    public Chromosome(){}
 
+    /**
+     * Create layer from adjacency matrix
```

Comment:
Loves the comments but make sure they do what they say ;).  It's alright to keep this if you will add this functionality later.